src/explore-education-statistics-common/src/modules/table-tool/components/utils/getCsvData.ts

```diff
@@ -65,13 +65,13 @@ export default function getCsvData(fullTable: WorkerFullTable): string[][] {
         });
 
         if (!matchingResult) {
-          return 'n/a';
+          return 'no data';
         }
 
-        return matchingResult.measures[indicator.value] ?? 'n/a';
+        return matchingResult.measures[indicator.value] ?? 'no data';
       });
 
-      if (indicatorCells.every(cell => cell === 'n/a')) {
+      if (indicatorCells.every(cell => cell === 'no data')) {
```

Comment:
Let's extract the 'no data' out into a variable like `TimePeriodDataTable` so we only need to change it in one place in the future.